sjb/config/test_cases/test_pull_request_origin_service_catalog.yml

```diff
@@ -0,0 +1,115 @@
+    - type: "script"
+      title: "build an origin release"
+      repository: "origin"
+      timeout: 5000
+      script: |-
+        if [[ -n "${PULL_REFS:-}" ]]; then
+          export OPENSHIFT_ANSIBLE_TARGET_BRANCH="${PULL_REFS%%:*}"
+        fi
+        git checkout ${OPENSHIFT_ANSIBLE_TARGET_BRANCH}
+        hack/build-base-images.sh
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local hack/env OS_ONLY_BUILD_PLATFORMS='linux/amd64' hack/build-rpm-release.sh
+        sudo systemctl restart docker
+        hack/build-images.sh
+        sed -i 's|go/src|data/src|' _output/local/releases/rpms/origin-local-release.repo
+        sudo cp _output/local/releases/rpms/origin-local-release.repo /etc/yum.repos.d/
+    - type: "script"
+      title: "expose the kubeconfig"
+      script: |-
+        sudo chmod a+x /etc/ /etc/origin/ /etc/origin/master/
+        sudo chmod a+rw /etc/origin/master/admin.kubeconfig
+    - type: "script"
+      title: "ensure built version of origin is installed"
+      timeout: 600
+      repository: "origin"
+      script: |-
+        origin_package="$( source hack/lib/init.sh; os::build::rpm::format_nvra )"
+        rpm -V "${origin_package}"
+    - type: "script"
+      title: "build service-catalog user-broker image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        REGISTRY=openshift/ NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog user-broker-image
+    - type: "script"
+      title: "build Service Catalog binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/update-generated-bindata.sh
+        cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/build-go.sh
+    - type: "script"
+      title: "build service catalog image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py service-catalog
+    - type: "script"
+      title: "build Service Catalog E2E test binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog  bin/e2e.test
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        oc cluster up --version=latest --service-catalog
+    - type: "script"
+      title: "Run Service Catalog E2E"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        oc login -u system:admin
+        SERVICECATALOGCONFIG=$KUBECONFIG cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/bin/e2e.test  -v 10 -alsologtostderr 
+  generated_artifacts:
+    origin_package_history.log: "sudo yum history info origin"
+    openshift_ansible_package_history.log: "sudo yum history info openshift-ansible"
+  system_journals:
```

Comment:
not useful

Reply:
I removed generated_artifacts, but left the bit about system_journals:

Reply:
Not sure `cluster up` will create any `systemd` services